mailbox/api/src/test/java/org/apache/james/mailbox/events/GroupContract.java

```diff
@@ -290,6 +328,16 @@ default void groupsDefinedOnlyOnSomeNodesShouldBeNotified() throws Exception {
             verify(mailboxListener, timeout(ONE_SECOND.toMillis()).times(1)).event(any());
         }
 
```

Comment:
Can we check one listener is called when we do a rediver in a distributed environment?